### PR TITLE
Improve Phlex test coverage

### PR DESCRIPTION
**Pull request by greenc-FNAL**
- **Improve test coverage of Phlex code**
- **Address https://github.com/Framework-R-D/phlex/pull/335#discussion_r2824494351**

### CODE REVIEW

File: phlex/model/identifier.hpp [hunk 1]
```diff
@@ -57,6 +57,14 @@ namespace phlex::experimental {
   inline std::string_view format_as(identifier const& id) { return std::string_view(id); }
 }
 
+template <>
```
Comment (beojan) [inline phlex/model/identifier.hpp]: This shouldn't be necessary. `format_as` is intended to handle this.

File: test/identifier.cpp [hunk 2]
```diff
@@ -65,5 +65,16 @@ int main()
     }
   }
   assert(ok);
+
+  // Additional coverage for identifier edge cases
+  identifier id1("abc");
+  identifier id2("def");
```
Comment (beojan) [inline test/identifier.cpp]: Comment needed about relative ordering, since it isn't alphabetical.

The comparison operators exist purely in order to make it possible to put identifiers into a `map` or `set`, not because they are inherently ordered.
